File: 프로그래머스/0/120886. A로 B 만들기/A로 B 만들기.java
import java.util.*;
class Solution {
    public int solution(String before, String after) {
        int answer = 0;
        
        for(int i = 0; i < before.length(); i++){
            char c = before.charAt(i);
            after = after.replaceFirst("" + c,"");
        }
        
        if(after.equals("")) answer++;
        
        return answer;
    }
}

File: 프로그래머스/0/120886. A로 B 만들기/A로 B 만들기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(const char* before, const char* after) {
    int answer = 1;
    int str[26] = {0, };
    int len = strlen(after);
    
    for(int i = 0; i < len; i++){
        str[before[i] - 97]++;
        str[after[i] - 97]--;
    }
    
    for(int j = 0; j < 26; j++){
        if(str[j] != 0){
            answer = 0;
            break;
        }
    }
    
    return answer;
}